arelle/plugin/validate/NL/rules/br_kvk.py

```diff
@@ -4,6 +4,7 @@
 from __future__ import annotations
 
 from datetime import date, timedelta
+from dateutil import relativedelta
```

Comment:
This looks like it needs to be explicitly required in pyproject.toml and requirements.txt